active-mq/values.yaml

```
@@ -159,14 +159,10 @@ volumeMounts:
-  #   command: ['sh', '-c', 'curl -L https://repo1.maven.org/maven2/org/jolokia/jolokia-war/1.7.2/jolokia-war-1.7.2.war -o /opt/jolokia/jolokia.war']
-  #   volumeMounts:
-  #   - name: jolokia-war
-  #     mountPath: /opt/jolokia
\ No newline at end of file
+
+config:
+  brokerXml:
+    global-max-size: 500Mb
+
+envVars:
+  JDK_JAVA_OPTIONS: -Xms512M -Xmx1G -Xlog:gc=debug:file=/tmp/gc.log
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)